code/modules/mob/living/silicon/ai/vox_sounds.dm

```diff
@@ -150,53 +188,79 @@ GLOBAL_LIST_INIT(vox_sounds, list("abortions" = 'sound/vox_fem/abortions.ogg',
 "condom" = 'sound/vox_fem/condom.ogg',
 "confirmed" = 'sound/vox_fem/confirmed.ogg',
 "connor" = 'sound/vox_fem/connor.ogg',
+"console2" = 'sound/vox_fem/console2.ogg',
+"console" = 'sound/vox_fem/console.ogg',

```

Comment:
Which one is which?  I imagine one is console as in device, and the other is console as in to provide emotional support.

Reply:
console = the machine
console2 = to care for someone